arcgis-ios-sdk-samples/Features/Feature layer rendering mode (map)/FeatureLayerRenderingModeMap.storyboard

```diff
@@ -13,37 +14,38 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="MHI-Dx-Wlm" userLabel="Dynamic Map View" customClass="AGSMapView">
-                                <rect key="frame" x="0.0" y="44" width="375" height="367"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="409"/>
                             </view>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="vR4-Yr-oi4" userLabel="Static Map View" customClass="AGSMapView">
-                                <rect key="frame" x="0.0" y="411" width="375" height="367"/>
+                                <rect key="frame" x="0.0" y="453" width="414" height="409"/>
                             </view>
-                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="EGP-7C-mcY">
-                                <rect key="frame" x="123" y="387" width="129" height="38"/>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="EGP-7C-mcY">
+                                <rect key="frame" x="142.5" y="429" width="129" height="38"/>
                                 <inset key="contentEdgeInsets" minX="10" minY="10" maxX="10" maxY="10"/>
                                 <state key="normal" title="Animated Zoom"/>
                                 <connections>
                                     <action selector="animateZoom:" destination="7OF-WW-ois" eventType="touchUpInside" id="lpu-EU-12z"/>
                                 </connections>
                             </button>
                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Static" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="aQ5-jH-GCE">
-                                <rect key="frame" x="20" y="431" width="44" height="21"/>
+                                <rect key="frame" x="20" y="473" width="44" height="21"/>
                                 <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                <nil key="textColor"/>
+                                <color key="textColor" white="0.0" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
```

Comment:
I believe this change is made on purpose, so no need to revert. 😳

Reply:
I think you're referring to the text color on a label. I the labels were set to `default` originally but I had to make them black because they turned white in dark mode and the labels became illegible.

Reply:
Yes! That makes sense.